.changeset/curvy-ants-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"eslint-plugin-svelte": minor
+---
+
+Add svelte/stores-no-async rule by @baseballyama in #225
```

Comment:
It's probably automatically linked. However, we've just introduced changesets, so I'm not sure if it works yet 😅 

```suggestion
Add svelte/stores-no-async rule
```

Reply:
fixed!
https://github.com/ota-meshi/eslint-plugin-svelte/pull/225/commits/715d04a1b1cc2cf013ae8462afa4f1f2a392c92b